admin_guide/out_of_resource_handling.adoc

```diff
@@ -64,8 +68,7 @@ Each threshold is of the following form:
 <eviction_signal><operator><quantity>
 ----
```

Comment:
this is basically describing a syntax for a threshold.

in 3.3, we just support literal thresholds (memory.available<100Mi)

in 3.4, we will support percentage thresholds as well (memory.available<10%)

so in 3.4, we will need to update this doc to reflect that both input styles are valid.

a sample node configuration is given in the "Example scenario" section.